server/src/test/scala/services/graphstages/ArrivalsGraphStagePaxNosSpec.scala

```diff
@@ -13,166 +13,122 @@ import services.{PcpArrival, SDate}
-  )
-
-  def pcpTimeCalc(a: Arrival): MilliDate = PcpArrival.pcpFrom(0, 0, _ => 0)(a)
-
-  "Given a live arrival with 0 pax that is not landing for more than 6 hours " +
-    "Then we should use the ACL pax numbers" >> {
-    val probe = TestProbe("arrivals")
-    val (aclSource, _, _, liveSource) = TestableArrivalsGraphStage(probe, buildArrivalsGraphStage).run
-
-    aclSource.offer(
-      List(arrival(actPax = Option(189), schDt = nowForThisTest.addHours(6).toISOString()))
-    )
-    liveSource.offer(
-      List(arrival(actPax = Option(0),
-        schDt = nowForThisTest.addHours(6).toISOString(),
-        gate = Option("G1")))
-    )
-
-    probe.fishForMessage(2 seconds) {
-      case ArrivalsDiff(toUpdate, _) =>
-        toUpdate.exists {
-          case (_, a) => a.ActPax == Option(189) && a.Gate == Option("G1")
-        }
+    "When I send an ACL arrival with zero pax" >> {
+      offerArrivalAndWait(crunch.baseArrivalsInput, scheduled = nowString, actPax = Option(0))
+      "I should see it with 0 ActPax in the port state" >> {
+        fishForArrivalWithActPax(Option(0))
+      }
     }
-    success
-  }
 
-  "Given a live arrival with 0 pax that is not landing for more than 6 hours with a forecast record " +
-    "Then we should use the Forecast pax numbers" >> {
-    val probe = TestProbe("arrivals")
-    val (aclSource, forecastSource, _, liveSource) = TestableArrivalsGraphStage(probe, buildArrivalsGraphStage).run
-
-    aclSource.offer(
-      List(arrival(actPax = Option(189), schDt = nowForThisTest.addHours(6).toISOString()))
-    )
-    forecastSource.offer(
-      List(arrival(actPax = Option(100), schDt = nowForThisTest.addHours(6).toISOString()))
-    )
-    liveSource.offer(
-      List(arrival(
-        actPax = Option(0),
-        schDt = nowForThisTest.addHours(6).toISOString(),
-        gate = Option("G1")))
-    )
-
-    probe.fishForMessage(2 seconds) {
-      case ArrivalsDiff(toUpdate, _) =>
-        toUpdate.exists {
-          case (_, a) => a.ActPax == Option(100) && a.Gate == Option("G1")
-        }
+    "When I send an ACL arrival with 100 pax" >> {
+      offerArrivalAndWait(crunch.baseArrivalsInput, scheduled = nowString, actPax = Option(100))
+      "I should see it with 100 ActPax in the port state" >> {
+        fishForArrivalWithActPax(Option(100))
+      }
     }
-    success
-  }
 
-  "Given a live arrival with 0 pax in the live feed that has landed " +
-    "Then we should assume that 0 is the correct number of pax regardless of forecast/acl numbers" >> {
-    val probe = TestProbe("arrivals")
-    val (aclSource, forecastSource, _, liveSource) = TestableArrivalsGraphStage(probe, buildArrivalsGraphStage).run
+    "When I send an ACL arrival with 100 pax and a forecast arrival with undefined pax" >> {
+      offerArrivalAndWait(crunch.baseArrivalsInput, scheduled = nowString, actPax = Option(100))
+      offerArrivalAndWait(crunch.forecastArrivalsInput, scheduled = nowString, actPax = None)
```

Comment:
could this test get a false positive by getting the first result through, and then ignoring the second once the forecast has been added?